### PR TITLE
Add Failure Screen Wails

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot from 2023-05-17 10-47-27

### CODE REVIEW

File: wails-frontend/src/pages/failure.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      case promptAction.deleteReq:
+        return 'The account could not been deleted';
+      case promptAction.newPasswordReq:
+        return 'The password could not be created';
+      case promptAction.importReq:
+        return 'The account could not be imported';
+      case promptAction.signReq:
+        return 'The transaction has been signed';
+      default:
+        return 'Apply';
```
Comment (mazmassa) [inline wails-frontend/src/pages/failure.tsx:20]: Is that correct have a default with Apply!? 🤔

**Review by mazmassa — APPROVED** [review]
Left a comment but LGTM!

File: wails-frontend/src/pages/failure.tsx (hunk 1 above)
Comment (pivilartisant) [reply wails-frontend/src/pages/failure.tsx:20]: no not correct will make changes

**Review by Thykof — APPROVED** [review]

File: wails-frontend/src/pages/failure.tsx (hunk 1 above)
Comment (pivilartisant) [reply wails-frontend/src/pages/failure.tsx]: chenged !

**Pull request merged**